schemas/com.shopify/update-shop/jsonschema/1-0-0

```diff
@@ -0,0 +1,166 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "com.shopify",
+    "name" : "update-shop",
+    "version" : "1-0-0",
+    "format" : "jsonschema"
+  },
+  "type" : "object",
+  "properties" : {
+    "has_discounts" : {
+      "type" : "boolean"
+    },
+    "city" : {
+      "type" : "string"
+    },
+    "name" : {
+      "type" : "string"
+    },
+    "setup_required" : {
+      "type" : "boolean"
+    },
+    "password_enabled" : {
+      "type" : "boolean"
+    },
+    "source" : {
+      "type" : "null"
+    },
+    "customer_email" : {
+      "type" : "null"
+    },
+    "force_ssl" : {
+      "type" : "boolean"
+    },
+    "latitude" : {
+      "type" : "number",
+      "minimum" : 0
+    },
+    "money_with_currency_in_emails_format" : {
+      "type" : "string"
+    },
+    "zip" : {
+      "type" : "string"
```

Comment:
Safe to have `maxLength`.

Reply:
Not sure here, what would it be? 10

Reply:
10 looks fine. Though I prefer "round" power-of-2-based numbers: 16, 32, etc.